2020-03-07 18:06:48,166 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 368823ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-07 19:10:59,326 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 3831143ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-07 19:32:05,633 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 1246155ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-07 19:52:44,757 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 1219114ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-07 20:13:42,987 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 1218119ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-07 20:29:16,529 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 913340ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-07 20:48:19,042 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 1102347ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-07 22:48:43,047 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 7203851ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-08 02:45:25,570 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 14202124ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-08 04:45:42,626 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 7216646ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-08 05:15:05,154 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 1762093ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-08 06:01:53,301 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 2808056ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-08 11:52:59,740 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 21066436ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-08 14:03:15,553 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 7214716ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-08 18:03:35,014 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 14419457ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-08 20:03:52,236 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 7217204ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-09 00:04:07,261 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 14415023ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-09 09:15:04,740 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 33057340ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-03-09 09:57:24,421 ERROR 56966 nodejs.ClusterClientNoResponseError: client no response in 2519570ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 56966
hostname: lixindeMacBook-Pro.local

2020-07-20 19:04:37,641 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 2409971ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-20 19:38:34,941 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 1997276ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-20 20:13:37,473 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 2082516ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-20 21:12:04,018 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 3446518ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-20 21:47:10,095 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 2066043ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-20 22:24:29,016 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 2198906ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-20 23:14:09,007 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 2959976ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-20 23:36:50,997 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 1321958ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 00:26:42,970 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 2951951ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 00:49:09,846 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 1306847ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 01:20:49,372 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 1858444ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 03:21:26,743 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 7217350ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 05:22:24,529 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 7217051ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 08:02:10,604 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 2288966ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 09:30:04,685 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 5254063ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 10:04:35,181 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 73584ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 10:40:29,941 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 694454ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 10:49:41,767 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 431794ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 10:58:09,303 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 107435ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 12:01:32,641 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 123471ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 12:13:43,291 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 490591ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 12:23:17,628 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 554321ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 12:44:50,024 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 1108645ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 13:16:32,975 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 1762406ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 13:48:25,743 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 1872397ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

2020-07-21 13:51:47,554 ERROR 27216 nodejs.ClusterClientNoResponseError: client no response in 141791ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/lixin/project/monorepo/packages/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 27216
hostname: lixindeMacBook-Pro.local

